components/timeline/TimelineDiscover.vue

```diff
@@ -6,10 +6,54 @@ const { locale: lang } = useI18n()
 const locale = getLanguageForRecs(lang.value)
 
 const recommendations: Recommendation[] = await $fetch(`/api/recommendations?locale=${locale}`)
+
+// Shorten a string to less than maxLen characters without truncating words.
+function shorten(str: string, maxLen: number): string {
+  if (str.length <= maxLen)
+    return str
+  return `${str.slice(0, str.lastIndexOf(' ', maxLen))}...`
+}
+
+// This is temporary untill we are able to fetch from the new endpoint
+function updateUTM(url: string): string {
+  return url.replace('pocket-newtab', 'mozilla')
+}
 </script>
 
 <template>
-  <div>
-    Coming soon...
-  </div>
+  <h1 text-2xl p-2>
+    Today’s Top Picks
+  </h1>
+  <main>
+    <template v-for="item in recommendations" :key="item.tileId">
+      <NuxtLink
+        :to="updateUTM(item.url)"
+        target="_blank"
+        external
+        style="padding: 16px 8px"
+        border="b base" flex
+      >
+        <div class="content" w-full pr>
+          <h4 text-sm text-secondary>
+            {{ item.publisher }}
+          </h4>
+          <h3 text-lg line-height-tight font-500 style="margin: 4px 0 4px">
```

Comment:
For this one, you should be able to use `m-y-4px` to get the 4px of margin on top and bottom. And then if you really need to force `0px` of margin on left/right, you could also use `m-x-0`.

Reply:
wowow Aly you are the UnoCSS expert 🥳

Reply:
Aly is getting all the UnoCSS questions in the future!